### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA announces additional inappropriate information when "Run/Debug Configurations" dialog is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1485]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Use shift+alt+f9 to open the list.
3. Tab to and select edit configurations.
4. "Run/Debug Configurations" dialog will open.
5. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA announces additional inappropriate information when "Run/Debug Configurations" dialog is opened 

**Jaws behaviour:**
Run/Debug configurations dialog
Debug - Azure Web App: untitled4:untitled4 dialog
Name JDK Web container Resource group Artifact Before launch: Launch Web Browser, Build Artifacts, Activate tool window 
Tree view
Azure Web App: untitled4:untitled4
1 of 1
To move through or expand items use the Arrow keys.

**NVDA Behaviour:**
Debug - Main  dialog  Before launch: Build, Activate tool window

Warning: No JDK specified for module 'untitled'
tree view
level 3  Main  selected  1 of 1

**Expected Result:**
Jaws and NVDA should announce only the dialog name when dialog is opened .
i.e Run/Debug configurations dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run-bebug configurations-reading entire content](https://user-images.githubusercontent.com/22956130/36106177-a21ee410-103c-11e8-8fce-a6f457eca265.jpg)
![11db63e5](https://user-images.githubusercontent.com/22956130/36106179-a56a7ad0-103c-11e8-8cfc-e0440b097dc6.jpg)

**Comment by Eskibear** [comment]
Content is provided by IntelliJ run/debug configuration, out of the toolkit's control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749959
